src/vs/workbench/contrib/notebook/browser/notebookOptions.ts

```diff
@@ -107,6 +107,7 @@ export interface NotebookOptionsChangeEvent {
 	readonly outputScrolling?: boolean;
 	readonly outputLinkifyFilePaths?: boolean;
 	readonly minimalError?: boolean;
+	readonly readonly?: boolean;
```

Comment:
@rebornix This seems to be the right fix, however
 there are issues related to this and my plan was to look into them in debt,
Here's a similar issue https://github.com/microsoft/vscode/issues/185320
Very likely related, I'd prefer to get this merged in debt, to avoid taking this in the last minute (last minute for me), 

Else if something goes wrong with this, then we end up with readonly notebooks which is a very bad bug.